### PR TITLE
MES-1045 - Health Declaration Rework

### PR DESCRIPTION
**Pull request by Cavalier07**
UI Rework for the Health Declaration Page

PO sign off given by Steve

unadjustednonraw_thumb_29

### CODE REVIEW

File: src/pages/health-declaration/health-declaration.html [hunk 1]
```diff
@@ -1,36 +1,31 @@
...
+      <div text-center class="candidate-driving-licence">{{getDriverNumber()}}</div>
+      <span class="mes-text-md decleration">You must declare any change to your health status since you last applied for a license.
+        It is a criminal
+        offense for you (or anyone else) to make a false statement in order for you to obtain a driving license and can
+        lead to prosecution.
       </span>
       <div>
         <input type="checkbox" name="health" id="confirm-health" [(ngModel)]="health-confirmation">
-        <label for="confirm-health">I confirm</label>
+        <label for="mes-header-sm">I confirm</label>
```
Comment (johnmorleybjss) [inline src/pages/health-declaration/health-declaration.html]: I don't think this `for` label is correct?

File: src/pages/health-declaration/health-declaration.html [hunk 2]
```diff
@@ -1,36 +1,31 @@
...
+      </div>
+      <div class="mes-divider"></div>
+      <div class="">
```
Comment (johnmorleybjss) [inline src/pages/health-declaration/health-declaration.html]: Empty class.

File: src/pages/health-declaration/health-declaration.html (hunk 1 above)
Comment (Cavalier07) [reply src/pages/health-declaration/health-declaration.html]: Thanks, have fixed and now i know why my styling didn't work

File: src/pages/health-declaration/health-declaration.html (hunk 2 above)
Comment (Cavalier07) [reply src/pages/health-declaration/health-declaration.html]: Fixed - Thanks 👍

File: src/pages/health-declaration/health-declaration.ts [hunk 3]
```diff
@@ -27,30 +26,8 @@ export class HealthDeclarationPage {
...
-
-  drawStart() {}
-
-  ionViewDidLoad() {
-    this.signaturePad.resizeCanvas();
-  }
-
   validation() {
-    if (this.confirmation && this.signature) {
+    if (this.confirmation && this.signaturePad.getSignature()) {
```
Comment (szabolcs-keresztes) [inline src/pages/health-declaration/health-declaration.ts]: Would be cleaner if you just returned the negated execution of the expression in the `if` statement.
But it's really just preference so it's up to you if you want to change it.

Comment (Cavalier07) [reply src/pages/health-declaration/health-declaration.ts]: Nice spot - Have updated

**Pull request merged**